unit/util/symbol_table.cpp

```diff
@@ -35,10 +35,11 @@ SCENARIO(
     symbol_tablet symbol_table;
 
     symbolt symbol;
-    irep_idt symbol_name = "Test";
+    irep_idt symbol_name = "Test_TestBase";
```

Comment:
This is because this was a test for something related added by @chrisr-diffblue but went in first, so my change had now broken on this test, so I had to edit this test to conform to what the validity checks expected, and make it pass again.